core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2Spec.scala

```diff
@@ -319,11 +321,72 @@ class BillingApiServiceV2Spec extends ApiServiceSpec with MockitoSugar {
         }
   }
 
+  it should "return 409 if adding an unregistered member during creation" in withEmptyDatabaseAndApiServices {
```

Comment:
Can you update the test description to make it clear that this is if `inviteUsersNotFound` is not true?

Reply:
Done in acf38584f